### DISCUSSION TITLE
Live Video Output Implementation

### DISCUSSION THREAD

**Discussion by VegaDeftwing** [noisedeck/noisedeck-docs#15] [Ideas]
For the electron (or alt framework) app that you have on the roadmap, it may be worth considering making the live view either be able to be popped out into its own window or act as a virtual webcam for the sake of streaming or capture in programs like OBS. Some applications only support webcams, so the virtual-webcam functionality may actually be useful even ontop of a separate render window.

**Comment by jesshewitt** [comment]
We do have this on the radar - I'll add it to the roadmap.

**Reply by j4s0n-c** [reply]
Spout on Windows and Syphon on Mac would be ideal for live input to "VJ" software. Using GPU memory to share the texture directly. Virtual webcam is basically universal, but most likely more CPU/RAM hungry. Canvas in a second window seems only useful for standalone or capturing with OBS or a second computer.

**Reply by jesshewitt** [reply]
Good info, thanks!
